gcloud/credentials.py

```diff
@@ -46,38 +46,51 @@ def get_credentials():
     """Gets credentials implicitly from the current environment.
 
     .. note::
-      You should not need to use this function directly. Instead, use the
-      helper method :func:`gcloud.datastore.__init__.get_connection`
-      which uses this method under the hood.
+
+        You should not need to use this function directly. Instead, use a
+        helper method which uses this method under the hood.
 
     Checks environment in order of precedence:
 
     * Google App Engine (production and testing)
-    * Environment variable GOOGLE_APPLICATION_CREDENTIALS pointing to
+    * Environment variable ``GOOGLE_APPLICATION_CREDENTIALS`` pointing to
```

Comment:
Maybe use the [`:envvar:` role](http://sphinx-doc.org/markup/inline.html#role-envvar)?